changedetectionio/notification.py

```diff
@@ -101,7 +101,9 @@ def process_notification(n_object, datastore):
                 apobj.notify(
                     title=n_title,
                     body=n_body,
-                    body_format=n_format)
+                    body_format=n_format,
+                    attach=None if not n_object['screenshot'] else n_object['screenshot']
```

Comment:
```suggestion
                    attach=n_object['screenshot']
```

Maybe apprise supports `False` in this case anyway?